charts/nocodb/values.yaml

```diff
@@ -86,6 +86,7 @@ extraSecretEnvs:
   NC_DB: "mysql2://mysql:3306?u=nocodb&p=secretPass&d=nocodb"
 
 storage:
+  enabled: true
```

Comment:
would recommend a comment which helps understand significance of this flag. 

 "# disabling this impacts the attachments functionality. 
 "# If disabled, recommend to configure NC_S3_BUCKET_NAME and other NC_S3* environment variables, refer documentation for more details"